reference/components/document-manager/using_the_document_manager.rst

```diff
@@ -62,6 +62,14 @@ on which subscribers you have registered.
 
 See the :doc:`subscribers` chapter for more information.
 
+.. note::
+
+    To create a document via a Sulu CLI command (e.g., import from an external source), run the command within the admin context.
+
+    .. code-block:: bash
+
+        bin/adminconsole your:command
```

Comment:
This text should be more general as it need to be the `admin` context. So you can not create a document inside a website controller you will need.

So something like "Documents can only be created in the admin context. For cli this means using the the `bin/adminconsole` and for controller it is required to have run them in the `/admin` context or using a symfony messenger to create them asynchron via admin CLI."

Reply:
```suggestion
    Be aware that documents can only be created in the admin context, not in the website context. For CLI commands, this means you must use ﻿``bin/adminconsole`` to execute the command. Alternatively, you can execute a command that dispatches messages via the Symfony Messenger. In this case, the consumer that consumes the dispatched message must also run in the admin context.

    .. code-block:: bash

        bin/adminconsole your:command
```